Library/Formula/exomizer.rb

```diff
@@ -0,0 +1,14 @@
+class Exomizer < Formula
+  desc     "Exomizer is a program that compresses files in a way that tries to be as efficient as possible but still allows them to be decompressed in environments where CPU speed and RAM are limited. For some popular 8-bit computers using 6502 compatible CPUs it can also generate executable files that decompress themselves in memory when run."
```

Comment:
Description is too long. Try running `brew audit --strict exomizer` to see style guidelines.